hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/HoodieDeltaStreamer.java

```diff
@@ -388,6 +388,9 @@ private boolean onDeltaSyncShutdown(boolean error) {
     @Parameter(names = {"--retry-last-pending-inline-clustering", "-rc"}, description = "Retry last pending inline clustering plan before writing to sink.")
     public Boolean retryLastPendingInlineClusteringJob = false;
 
+    @Parameter(names = {"--fetch-from-multi-sources"}, description = "Fetch data from multi sources and ingest to single sink table")
```

Comment:
nit: in the description, lets change multi to multiple.

Reply:
ok, I've modified it.